app/config/routing_dev.yml

```diff
@@ -1,15 +1,14 @@
 _wdt:
-    resource: "@WebProfilerBundle/Resources/config/routing/wdt.xml"
+    resource: '@WebProfilerBundle/Resources/config/routing/wdt.xml'
```

Comment:
Do we have code conventions for quotes in yaml files?

And in js and html? The latter one I ask because we started to use " there.

Reply:
There's no convention. I used single quotes here because Symfony does (simple copy&paste), but in general quotes are semantically similar as in PHP, see http://yaml.org/spec/1.2/spec.html#id2786942.